components/brave_wallet_ui/utils/address-utils.ts

```diff
@@ -4,9 +4,17 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import {
+  BraveWallet,
   WalletAccountType
 } from '../constants/types'
 
+export function isValidFilAddress (value: string): boolean {
+  if (!value.startsWith(BraveWallet.FILECOIN_MAINNET) && !value.startsWith(BraveWallet.FILECOIN_TESTNET)) {
+    return false
+  }
+  return (value.length === 41 || value.length === 86)
```

Comment:
Can you please add some comments about these magic numbers? Ex, which one corresponds to secp256k1 address, vs BLS.